playbooks/roles/shadowsocks/templates/instructions.md.j2

```diff
@@ -64,6 +64,17 @@ For reference, this is the configuration specified:
         Password: {{ shadowsocks_password.stdout }}
         Encryption Method: {{ shadowsocks_encryption_method }}
 
+For users on unreliable or hostile networks (esp. experiencing quality-of-service (QOS) throttling), further configuration of Shadowsocks to use the simple-obfs plugin can be carried out via the following configuration:
```

Comment:
What do you think about adding this text at the very bottom of the file under its own heading instead of under the `Linux` heading. Something like:

```
 <a name="simple-obfs"></a>
 ### simple-obfs for unreliable/hostile networks ###
```

I think this information might be useful to folks other than Linux users.